packages/cli/src/api-gh.ts

```diff
@@ -0,0 +1,118 @@
+import { fetch, $ } from 'zx';
+import libsodium from 'libsodium-wrappers';
+
+export default async (
+  storeId: number,
+  ecomApiKey: string,
+  ecomAuthentication: string,
+  firebaseServiceAccount: string | null,
+  ghToken: string,
```

Comment:
```suggestion
  ghToken: string,
  ghRepo?: string,
```

Quando automatizarmos o setup da loja (pra SaaS) vamos ter que executar essa função dentro de uma aplicação, sem CLI, aí teria que passar o `ghRepo` (gerado automaticamente antes de chegar aqui) porque não rola executar o comando do git pra buscar essa info.

Reply:
Por esse mesmo motivo essa função aqui https://github.com/ecomplus/cloud-commerce/blob/main/packages/cli/src/create-auth.ts tá isolada do [login em CLI](https://github.com/ecomplus/cloud-commerce/blob/main/packages/cli/src/login.ts) 🤙🏾